docs/source/sediment_formulation_2d.rst

```diff
@@ -0,0 +1,66 @@
+2D sediment transport formulation
+=================================
+
+.. highlight:: python
+
+Governing equations
+-------------------
+
+Suspended sediment transport is modelled in two dimensions
+using an advection-diffusion equation
+:eq:`sediment_eq_2d`.
+If solved in non-conservative form, the prognostic variable
+is the passive tracer concentration,
+:math:`T`. The corresponding field in Thetis is called
+``'sediment_2d'``.
+
+A conservative suspended sediment transport model is also
+available. In this case, the equation is solved for
+:math:`q=HT`, where
+:math:`H` is the total water depth.
+The conservative tracer model is specified using the
+:ref:`ModelOptions2d<model_options_2d>`.\ :py:attr:`.use_sediment_conservative_form`
+option.
+
+Bedload transport is modelled in two dimensions using the
+Exner equation
+:eq:`exner_eq`.
+It is solved for the bedlevel,
+:math:`z_b`, which has the effect of modifying the bathymetry.
```

Comment:
I would say ":math:`z_b` which is the bathymetry" because if you look at the code z_b is the bathymetry